src/main/views/macro/interestSnippet.njk

```diff
@@ -1,23 +1,29 @@
 {% macro interestSnippet(label = 'Interest', rate, numberOfDays, interestDate, interestToDate, specificDailyAmount) %}
-  <details role="group">
+  {% if rate or specificDailyAmount > 0 %}
+    <details role="group">
     <summary role="button" aria-controls="details-content-0" aria-expanded="false">
       {{ t(label) }}
     </summary>
-    <div class="panel text-small" id="details-content-0" aria-hidden>
-      <div>
-        {% if rate %}
-        {{ t('Interest calculated at %s%% for %s days', {
-          postProcess: 'sprintf', sprintf: [ rate, numberOfDays ] })
-        }}
-        {% else %}
-          {{ t('Interest calculated with daily interest amount of %s for %s days', {
-            postProcess: 'sprintf', sprintf: [ specificDailyAmount | numeral, numberOfDays ] })
-          }}
-        {% endif %}
+    {% if rate %}
+      <div class="panel text-small" id="details-content-0" aria-hidden>
+        <div>
+          {{ t('Interest calculated at %s%% for %s days', {
+            postProcess: 'sprintf', sprintf: [ rate, numberOfDays ] }) }}
+          ({{ t('%s to %s', { postProcess: 'sprintf', sprintf: [ interestDate | date, interestToDate | date ]}) }})
+        </div>
       </div>
-      <div class="font-xsmall">
-        ({{ t('%s to %s', { postProcess: 'sprintf', sprintf: [ interestDate | date, interestToDate | date ]}) }})
+    {% elseif specificDailyAmount > 0 %}
+      <div class="panel text-small" id="details-content-0" aria-hidden>
+        <div>
+          {{ t('Interest calculated with daily interest amount of %s for %s days', {
```

Comment:
Please change it to use new approach. Example can be found here:
src/main/features/dashboard/views/claimant.njk:22

Reply:
done